src/main/java/pl/stormit/tinyurl/dto/UrlAnalyticsDto.java

```diff
@@ -0,0 +1,24 @@
+package pl.stormit.tinyurl.dto;
+
+import lombok.Data;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import java.time.Instant;
+
+@Data
+public class UrlAnalyticsDto {
+
+    @NotBlank
```

Comment:
NotNull?

Reply:
I didn't used @NotNull, because this constraint won't allow null values for the totalClicks field, but the field can be empty.
I would like this field can't be empty and null values.
Maybe better use @NotEmpty ?

Reply:
Check my comment on slack

Reply:
Improved